.codespell/wordlist.txt

```diff
@@ -4,3 +4,4 @@ ue
 tha
 bu
 crate
+DNE
```

Comment:
what is it?

Reply:
does not exist but i can update it to be more explicit